clusters/nerc-ocp-prod/ope-autograder/application.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: ope-autograder
+  labels:
+    nerc.mghpcc.org/sync-policy: common
+spec:
+  destination:
+    name: nerc-ocp-prod
+    namespace: ece440spring2024-619f12
+  project: default
+  source:
+    repoURL: https://github.com/IsaiahStapleton/ope-autograder-nerc-app.git
+    path: manifests
```

Comment:
@IsaiahStapleton I don't see a `manifests` directory in the repo. Also, should we pull in this `ope-autograder-nerc-app` into the `OCP-on-NERC` org if it's going to be an Application in NERC?

Reply:
I changed it to the correct path. As for ope-autograder-nerc-app, I talked to Lars before and I believe he said it would be fine as an external repo. @larsks is that correct? Or do you think we should put it in OCP-on-NERC?